WordPress/gutenberg
# Packages: Adding RTL CSS support to the packages CSS

This PR adds RTL support to the packages CSS which allows us to drop the root package folders for all the packages that expose CSS files.

There's a new Webpack step copying the built CSS files from the packages folder into the build folder used by the WordPress registered styles.

**Testing instructions**

 - Check that the `nux` RTL CSS file is generated properly in `build-style` and copied to the global `build` folder.
